### DISCUSSION TITLE
Ignore empty lines in region

### DISCUSSION THREAD

**Discussion by jgollenz** [numToStr/Comment.nvim#214] [Q&A] [answered]
Often times, I toggle off groups of consecutive statements. Those groups are each separated by empty lines. Then when I want to toggle on all of the groups. This does not work, because of the empty lines between them: 

![block-comments](https://user-images.githubusercontent.com/12050098/185795812-0e5bff4f-7032-429d-bbb4-d114e94deb38.gif)

I cannot think of a situation where the current behavior is the one I would want, but I might be missing something. Perhaps this is already possible with the plugin, but I don't know how. Any thoughts?

**Comment by numToStr** [comment, marked as answer]
Why not ignore empty lines completely?

```lua
require'Comment'.setup({ ignore = '^$' })
```

Please read: https://github.com/numToStr/Comment.nvim#-ignoring-lines

**Comment by numToStr** [comment]
(Also please use https://github.com/numToStr/Comment.nvim/discussions for asking questions :) )

**Comment by jgollenz** [comment]
Shame on me. Sorry for wasting your time. Thanks for the quick reply. Will ask in discussions *after* reading the docs in the future.
